controllers/users.js

```diff
@@ -1,48 +1,52 @@
-const { v4: uuidv4 } = require('uuid');
-const userModel = require('../db/user.model');
+const User = require('../models/user');
+const {
+    Exception
+} = require('../exceptions/Exception');
 const { 
+    UserValidationException,
     UserAlreadyExistsException, 
     UserNotFoundException 
 } = require('../exceptions/userExceptions');
+const {
+    DatabaseException
+} = require('../exceptions/databaseExceptions');
 
 class UsersController {
-    static addUser(username, password) {
-        if(userModel.find({ username }).length > 0) {
+    static async addUser(username, password) {
+        if(await User.count({ where: { username: username } })) {
             throw new UserAlreadyExistsException();
         }
-        const user = userModel.build({ 
-            id: uuidv4(), 
+        const user = User.build({
```

Comment:
Don't use user.build and user.save, the `controller` should only call 1 function, and you can have that logic inside that function, which will be `services` unction